### PR TITLE
add flake8-bugbear

### PR DESCRIPTION
**Pull request by iKintosh**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Type of Change

- [x] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by Mr-Geekman** [comment]
Should we fix lint test phase before review?

**Comment by iKintosh** [comment]
> Should we fix lint test phase before review?

I've done it on purpose. This will allow us to see if the checks are necessary to the project.

**Comment by iKintosh** [comment]
Removed mutable, since bugbear have the same functionality;

File: etna/analysis/plotters.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def plot_forecast(
 
         # plot forecast plot for each of given forecasts
         quantile_prefix = "target_"
-        for j, (forecast_name, forecast) in enumerate(forecast_results.items()):
+        for _j, (forecast_name, forecast) in enumerate(forecast_results.items()):
```
Comment (Mr-Geekman) [inline etna/analysis/plotters.py:180]: Why?

Comment (iKintosh) [reply etna/analysis/plotters.py:180]: `B007: Loop control variable not used within the loop body. If this is intended, start the name with an underscore.`
https://github.com/PyCQA/flake8-bugbear

Comment (Mr-Geekman) [reply etna/analysis/plotters.py:180]: Should we use in that case:
```python
for forecast_name, forecast in forecast_results.items():
```

Comment (iKintosh) [reply etna/analysis/plotters.py:180]: I agree that it will have more sense and change it in the next commit.

Comment (iKintosh) [reply etna/analysis/plotters.py]: Fixed

**Review by Mr-Geekman — APPROVED** [review]

**Pull request merged**